### PR TITLE
Onboarding

### PR DESCRIPTION
**Pull request by mehlium**
Added a proper flow for onboarding. Placeholder texts and values so far! Let me know what you think.

### CODE REVIEW

**Comment by Citrullin** [comment]
1. Okay, I upgraded the wallet and it gives me a wrong pin error, while the pin is definitely right. 4 times zero. This is worth discovering in the future. See, if this happens again.
2. Once I generated a new wallet, I am not able to scroll down to see the actual seed. this worked before. Maybe related to #44 
3. I've backed up my seed on the right side is too long.
4. Clicking on I've backed up my seed gets you into the main menu.

**Comment by mehlium** [comment]
Did not think of how to "upgrade" an existing wallet, this is something we have to be careful with in the future (once the implementation stabilizes it won't be much of a problem). Fixed some of the apparently broken things with the PR, left a few things as is so it does not grow even more:

* Scrolling seed (or other views) should be fixed in #23 or #44 
* When setting an alias / pin it's no way to edit, should be fixed in #40

**Comment by Citrullin** [comment]
- "Create New" needs to be shorter. It looks like there is only one option. It probably has to be just "New". 
- Scrolling still doesn't work. 
- There is a horizontal scrolling bar in the wallet. Wasn't there before. You cannot scroll it. So, just a minor issue.
- When adding account, it probably makes sense to ask for a name there as well. But we can make a separate issue for that. 
- When receiving funds and updating the wallet with the update button, the amount in the top doesn't get updates, but the transactions are in the transaction history.
- When I am in the setting of changing the account name, I am not able to go back. Guess this is related to #40 

We can also make separated issues for that and just merge it. I take a look into the code then.

File: public/locales/en-US.properties [hunk 1]
```diff
@@ -77,3 +70,31 @@ unlocking-wallet = Unlocks..
 adding-account = Adds new account..
 loading-refresh = Loads transactions..
 sending-funds = Sending funds..
+
+# Onboard
+onboard-button-back = Back
```
Comment (Citrullin) [inline public/locales/en-US.properties:75]: Do we want to use a prefix for everything? Like onboard__ and create__ or something similar?

File: src/App.svelte [hunk 2]
```diff
@@ -1,14 +1,17 @@
...
 	import Setup from "./view/Setup.svelte";
 	import About from "./view/About.svelte";
 	import Wallet from "./view/Wallet.svelte";
 	import UnlockWallet from "./view/UnlockWallet.svelte";
-	import CreateWallet from "./view/CreateWallet.svelte";
 	import SoftwareKeys from "./view/SoftwareKeys.svelte";
 	import Toast from "./components/Toast.svelte";
+	import Onboard from "./view/Onboard.svelte";
+	import {afterUpdate, onMount} from "svelte";
+	import { walletExists } from "./machinery/secure-storage";
```
Comment (Citrullin) [inline src/App.svelte]: Inconsistent to the other imports. Should be `import {walletExists} from "./machinery/secure-storage";`, I guess.

File: src/machinery/eventListener.ts [hunk 3]
```diff
@@ -22,18 +26,20 @@ const elementSelector = (selectedElement) => {
 let navigation = new Navigation([], elementSelector);
 
 loadedComponentStore.subscribe((value) => {
-  if (value.elements.length > 0) {
```
Comment (Citrullin) [inline src/machinery/eventListener.ts:25]: Why did you remove it? If the elements are empty, it shouldn't create a navigation and attach keydown listener to it.

File: src/machinery/wallet.ts [hunk 4]
```diff
@@ -1,38 +1,43 @@
...
 
-export function generateWallet(): WalletResult {
-  const w = wallet.generate();
-  return {
-    seed: w.seed,
-    mnemonic: w.mnemonic,
-  };
+export function generateWallet(): Promise<WalletResult> {
+  return new Promise((resolve) => {
+    setTimeout(() => {
```
Comment (Citrullin) [inline src/machinery/wallet.ts:12]: What is this setTimeout for?

File: src/view/onboard/CreateSeed.svelte [hunk 5]
```diff
@@ -0,0 +1,41 @@
+<script lang="ts">
+    import Text from "../../components/Text.svelte";
+    import {onMount} from "svelte";
+    import type {WalletResult} from "../../machinery/wallet";
+    import { generateWallet } from "../../machinery/wallet";
```
Comment (Citrullin) [inline src/view/onboard/CreateSeed.svelte]: Same here. Inconsistent

File: public/locales/en-US.properties (hunk 1 above)
Comment (mehlium) [reply public/locales/en-US.properties:75]: I think it can be smart so that we know the context of the text. However, when we finish with most of the text it's probably easier to see what are the common things, and what we might need context for (if any).

File: src/machinery/eventListener.ts (hunk 3 above)
Comment (mehlium) [reply src/machinery/eventListener.ts:25]: Because we need to initialize this to set/unset selection and navigation on view change. So even though elements are an empty array, we need to know that in order to do right thing in for instance "Enter" case.

File: src/machinery/wallet.ts (hunk 4 above)
Comment (mehlium) [reply src/machinery/wallet.ts:12]: To be able to delay the function, so that we can show the loading screen. There might be a more ideomatic way to do this though..

**Comment by mehlium** [comment]
Fixed the inconsistencies, and added a few of the unknown issues as new issues. Thanks for testing!
